### PR TITLE
trying to fix hight CPU following this issue: https://github.com/jits…

### PR DESCRIPTION
**Pull request by juliemar**
### Description
Some Computers presents high CPU during the meetings. [this](https://github.com/jitsi/jitsi-meet/issues/2596) issue explain alternatives and I applied in this PR following [this](https://github.com/jitsi/jitsi-meet/blob/master/interface_config.js) and [this](https://github.com/jitsi/jitsi-meet/blob/master/config.js)

### How to test?
Running in localhost when you open a meet and click with the right button in the iframe the meet.jitsi.org URL needs to be like this "https://meet.jit.si/6c04907e-f402-4b63-8b9e-b6f324e13166-localhost#jitsi_meet_external_api_id=1&config.resolution=360&config.constraints=%7B%22video%22%3A%7B%22ideal%22%3A360%2C%22max%22%3A360%7D%7D&interfaceConfig.DISABLE_VIDEO_BACKGROUND=true&interfaceConfig.TOOLBAR_BUTTONS=%5B%22microphone%22%2C%22camera%22%2C%22closedcaptions%22%2C%22desktop%22%2C%22fullscreen%22%2C%22fodeviceselection%22%2C%22hangup%22%2C%22profile%22%2C%22etherpad%22%2C%22sharedvideo%22%2C%22settings%22%2C%22raisehand%22%2C%22videoquality%22%2C%22filmstrip%22%2C%22stats%22%2C%22shortcuts%22%2C%22tileview%22%2C%22chat%22%5D"   

### Expected behavior
Expected the CPU utilization less then 30%

### CODE REVIEW

File: frontend/office.js [hunk 1]
```diff
@@ -119,7 +119,18 @@ $(() => {
       width: "100%",
       height: "100%",
       parentNode: document.querySelector("#meet"),
+      configOverwrite: {
+        //startAudioOnly:true,
+        resolution :360,
```
Comment (alesshh) [inline frontend/office.js]: lint it 💄

File: frontend/office.js [hunk 2]
```diff
@@ -119,7 +119,18 @@ $(() => {
...
+        constraints: {
+          video: {
+            ideal: 360,
+            max:360
+          }
+        }   
+      },
       interfaceConfigOverwrite: {
+        DISABLE_VIDEO_BACKGROUND:true,
```
Comment (alesshh) [inline frontend/office.js]: lint it 💅

Comment (juliemar) [reply frontend/office.js]: could you update my branch with this lint? :)

File: frontend/office.js (hunk 1 above)
Comment (juliemar) [reply frontend/office.js]: could you update my branch with this lint? :)

**Pull request merged**